README.md

```diff
@@ -75,34 +75,61 @@ The inventory service is an independent web service that accepts orders via HTTP
 
 ### Developing the Ballerina services with circuit breaker
 
-The `ballerina/http` package contains the circuit breaker implementation. After importing that package you can create a client with a circuit breaker. The `endpoint` keyword in Ballerina refers to a connection with a remote service. The following code segment is adding the circuit breaker capabilities for the endpoint.
+The `ballerina/http` package contains the circuit breaker implementation. After importing that package you can create
+a client with a circuit breaker. `http:Client` allows establishing a connection with a remote party (endpoint), circuit breaker configurations could be provided while initializing the client. 
+The following code segment will add the circuit breaker capabilities for the endpoint.
 
 ```ballerina
-endpoint http:Client circuitBreakerEP {
-
-    // The 'circuitBreaker' term incorporate circuit breaker pattern to the client endpoint
-    // Circuit breaker will immediately drop remote calls if the endpoint exceeded the failure threshold
-    circuitBreaker: {
-        // Failure calculation window.
-        rollingWindow: {
-            // Time period in milliseconds for which the failure threshold is calculated.
-            timeWindowMillis: 10000,
-            // The granularity at which the time window slides. This is measured in milliseconds.
-            bucketSizeMillis: 2000,
-            // Minimum number of requests in a `RollingWindow` that will trip the circuit.
-            requestVolumeThreshold: 0
+http:Client circuitBreakerEP = new("http://localhost:9092", config = {
+        // The 'circuitBreaker' term incorporate circuit breaker pattern to the client endpoint
+        // Circuit breaker will immediately drop remote calls if the endpoint exceeded the failure threshold
+        circuitBreaker: {
+            // Failure calculation window. This is how long Ballerina
+            // circuit breaker keeps the statistics for the operations.
+            rollingWindow: {
+
+                // Time period in milliseconds for which the failure threshold
+                // is calculated.
+                timeWindowMillis: 10000,
+
+                // The granularity at which the time window slides.
+                // This is measured in milliseconds.
+                // The `RollingWindow` is divided into buckets
+                //  and slides by these increments.
+                // For example, if this `timeWindowMillis` is set to
```

Comment:
Let's merge this into one sentence. Something like:

// For example, if this `timeWindowMillis` is set to
// 10000 milliseconds and `bucketSizeMillis` is set to 2000,
// `RollingWindow` breaks into sub windows with
// 2-second buckets and stats are collected with
// respect to the buckets